src/components/ConfirmLeavingPage/ConfirmUnsavedModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button, Modal } from '@grafana/ui';
+
+import { DataTestIds } from '../../test/dataTestIds';
+
+interface ConfirmUnsavedModalProps {
+  onLeavePage: () => void;
+  onStayOnPage: () => void;
+}
+
+export function ConfirmUnsavedModal({ onLeavePage, onStayOnPage }: ConfirmUnsavedModalProps) {
+  return (
+    <Modal isOpen title="Unsaved changes" onDismiss={onStayOnPage}>
+      <h5 data-testid={DataTestIds.CONFIRM_UNSAVED_MODAL_HEADING}>
```

Comment:
I have no doubt you copied this from somewhere else so isn't on you but it is weird this is an h5. Can it just be a normal div or p tag and then be made to look the same as whatever the design system recommends?